README.md

```diff
@@ -1 +1,682 @@
+        }else{    
+            //ignoring invalid orderTypes  
+        log:printInfo("No any valid order type recieved, ignoring the message, order type recieved - " + orderType.toString());
+        }    
+    }
+}
+```
+
+**retail_order_process_service.bal**
+
+```
+import ballerina/log;
+import ballerina/jms;
+
+
+// Initialize a JMS connection with the provider
+// 'Apache ActiveMQ' has been used as the message broker
+jms:Connection conn = new({
+        initialContextFactory: "org.apache.activemq.jndi.ActiveMQInitialContextFactory",
+        providerUrl: "tcp://localhost:61616"
+    });
+
+// Initialize a JMS session on top of the created connection
+jms:Session jmsSession = new(conn, {
+        // Optional property. Defaults to AUTO_ACKNOWLEDGE
+        acknowledgementMode: "AUTO_ACKNOWLEDGE"
+    });
+
+// Initialize a retail queue receiver using the created session
+endpoint jms:QueueReceiver jmsConsumer {
+    session:jmsSession,
+    queueName:"Retail_Queue"
+};
+
+// JMS service that consumes messages from the JMS queue
+// Bind the created consumer to the listener service
+service<jms:Consumer> orderDispatcherService bind jmsConsumer {
+    // Triggered whenever an order is added to the 'Order_Queue'
+    onMessage(endpoint consumer, jms:Message message) {
+
+        log:printInfo("New order received from the JMS Queue");
+        // Retrieve the string payload using native function
+        var orderDetails = check message.getTextMessageContent();
+        log:printInfo("below retail order has been successfully processed");
+        log:printInfo(orderDetails);
+    }
+}
+
+```
+
+**wholesale_order_process_service.bal**
+
+```
+import ballerina/log;
+import ballerina/jms;
+
+
+// Initialize a JMS connection with the provider
+// 'Apache ActiveMQ' has been used as the message broker
+jms:Connection conn = new({
+        initialContextFactory: "org.apache.activemq.jndi.ActiveMQInitialContextFactory",
+        providerUrl: "tcp://localhost:61616"
+    });
+
+// Initialize a JMS session on top of the created connection
+jms:Session jmsSession = new(conn, {
+        // Optional property. Defaults to AUTO_ACKNOWLEDGE
+        acknowledgementMode: "AUTO_ACKNOWLEDGE"
+    });
+
+// Initialize a retail queue receiver using the created session
+endpoint jms:QueueReceiver jmsConsumer {
+    session:jmsSession,
+    queueName:"Wholesale_Queue"
+};
+
+// JMS service that consumes messages from the JMS queue
+// Bind the created consumer to the listener service
+service<jms:Consumer> orderDispatcherService bind jmsConsumer {
+    // Triggered whenever an order is added to the 'Order_Queue'
+    onMessage(endpoint consumer, jms:Message message) {
+
+        log:printInfo("New order received from the JMS Queue");
+        // Retrieve the string payload using native function
+        var orderDetails = check message.getTextMessageContent();
+        log:printInfo("below wholesale order has been successfully processed");
```

Comment:
Did we check whether the json path working when we retrieve jms object, it was not working ballerina 0.75x. versions, can we double check this? please retrieve order information and print in this service. If not working please escalate to the product team